common/datavalues/src/arrays/ops/contain.rs

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020-2021 The Datafuse Authors.
+//
+// SPDX-License-Identifier: Apache-2.0.
+
+use std::fmt::Debug;
+
+use common_arrow::arrow::array::ArrayRef;
+use common_arrow::arrow::compute::contains::contains;
+use common_exception::ErrorCode;
+use common_exception::Result;
+
+use crate::arrays::get_list_builder;
+use crate::arrays::DataArray;
+use crate::prelude::*;
+use crate::series::IntoSeries;
+
+pub trait ArrayContain: Debug {
+    /// # Safety
+    ///
+    fn contain(&self, _list: &DFListArray) -> Result<DFBooleanArray>
+    where Self: std::marker::Sized {
+        Err(ErrorCode::BadDataValueType(format!(
+            "Unsupported apply contain operation for {:?}",
+            self,
+        )))
+    }
+}
+
+macro_rules! contain_internal {
+    ($self:expr, $list:expr) => {{
+        let arrow_array = $self.downcast_ref();
+
+        // List's length must be 1. Need to create
+        assert_eq!($list.len(), 1);
```

Comment:
Why list's size must be 1 ?

Reply:
ListArray is 

```
[
[1,2,3],
[4,5],
[6]
]
```

Array is 
```
[
  1,
  5,
  7 
]
```

`contains(list, array)` ---> `[true, true, false]`

Reply:
I see. I renamed this API to contain_inlist(self, &Series) and added another API called contain which calls arrow2 API directly. 
Please take a look at the PR again.